docs/sources/project/review-pr.md

```diff
@@ -49,15 +49,22 @@ need to update your pull request with additional changes.
 
 To update your existing pull request:
 
-1. Change one or more files in your local `docker-fork` repository.
+1. Checkout the PR branch in your local `docker-fork` repository.  
 
-2. Commit the change with the `git commit --amend` command.
+    This is the branch associated with your request.
+
+2. Change one or more files and then stage your changes using the
+   command `git add`.
+
+    	$ git add <path_or_filename>
```

Comment:
We use $ to indicate an actual command line.  That isn't the case here. This is a syntax example.  Can you remove that please?